TextToSpeech/impl/TTSManager.cpp

```diff
@@ -215,7 +221,15 @@ TTS_Error TTSManager::speak(int speechId, std::string text) {
 
     if(m_speaker) {
         // TODO: Currently 'secure' is set to true. Need to decide about this variable while Resident app integration.
-        m_speaker->speak(this, speechId , text, true,m_defaultConfiguration.primVolDuck());
+        if(m_defaultConfiguration.checkAccess("speak",callsign))
```

Comment:
Wherever applicable, please leave a space after the coma operator to improve readability

Reply:
ok